config/secrets.defaults.yml

```
@@ -0,0 +1,14 @@
+---
+test:
+  amazon_defaults: &amazon_defaults
+    client_id: AMAZON_CLIENT_ID
+    client_secret: AMAZON_CLIENT_SECRET
+  amazon_eks_defaults: &amazon_eks_defaults
+    hostname: hostname
```

Concerned that "hostname" might be too generic...should we do:

```suggestion
    hostname: amazonhostname
```

---

Ok, done